src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner input = new Scanner(System.in);
+        printStartMessage();
+
+        String userInput;
+        Task [] tasks = new Task [100];
+        int numTasks = 0;
+
+        while (true) {
+            // Get and parse user input
+            userInput = input.nextLine();
+            // Exit if user says bye
+            if (userInput.equals("bye")) {
+                break;
+            }
+            String[] parsedInput = userInput.split(" ", 2);
+            String command = parsedInput[0];
+            String words = "";
+            if (parsedInput.length != 1) {
+                words = parsedInput[1];
+            }
+
+            switch(command) {
+            // List all the tasks
+            case "list":
+                listTasks(tasks, numTasks);
+                break;
+            // Mark a task as done
```

Comment:
The comment should be written on the same line as the function to avoid confusion between cases